pkg/updater/install.go

```diff
@@ -96,19 +96,27 @@ func (i *installer) uninstallExperiment(pkg string) error {
 }
 
 func (i *installer) startExperiment(pkg string) error {
-	// TODO(arthur): currently we only support the datadog-agent package
-	if pkg != "datadog-agent" {
+	switch pkg {
```

Comment:
you will need to set cap_setuid to the helper file on the newly installed updater before triggering it